arduino/smartcar/smartcar.ino

```diff
@@ -1,9 +1,66 @@
 #include <Smartcar.h>
 
-void setup() {
-  // put your setup code here, to run once:
-}
+//only for printing current readings of sensor to serial terminal
+const unsigned long PRINT_INTERVAL = 100;
+unsigned long previousPrintout     = 0;
+
+ArduinoRuntime arduinoRuntime;
+BrushedMotor leftMotor(arduinoRuntime, smartcarlib::pins::v2::leftMotorPins);
+BrushedMotor rightMotor(arduinoRuntime, smartcarlib::pins::v2::rightMotorPins);
+DifferentialControl control(leftMotor, rightMotor);
+
+GY50 gyroscope(arduinoRuntime, 37);
+
+const auto pulsesPerMeter = 600;
+
+DirectionlessOdometer leftOdometer{
+    arduinoRuntime,
+    smartcarlib::pins::v2::leftOdometerPin,
+    []() { leftOdometer.update(); },
+    pulsesPerMeter};
+DirectionlessOdometer rightOdometer{
+    arduinoRuntime,
+    smartcarlib::pins::v2::rightOdometerPin,
+    []() { rightOdometer.update(); },
+    pulsesPerMeter};
+
+
 
-void loop() {
-  // put your main code here, to run repeatedly:
+SmartCar car(arduinoRuntime, control, gyroscope, leftOdometer, rightOdometer);
+
+const auto triggerPin = 6;
+const auto echoPin = 7;
+const auto maxDistance = 400;
```

Comment:
We should keep naming consistent. If you've used uppercase for constants already, then these should follow that naming convention as well.